src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -26,14 +26,21 @@
           Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'"
           Inputs="$(StringResourcesPath)"
           Outputs="$(IntermediateResOutputFileFullPath)">
-    
-    
 
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
         AssemblyName="$(AssemblyName)" />
 
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true'"
+          SourceFiles="$(StringResourcesPath)"
+          DestinationFiles="$(ResourcesFolderPath)\FxResources.$(AssemblyName).SR.resw"
+          SkipUnchangedFiles="true" />
+
+    <Delete Condition="'$(BuildingUAPVertical)' == 'true' AND Exists('$(CommonPriFile)')"
```

Comment:
Fixed. I didn't remove the CommonPriFile property definition as we will be using it later in test.targets. Should I move it to dir.props instead?